### ISSUE TITLE
watch and computed sometimes not detecting

### ISSUE THREAD

**Issue by lexar19** [coders-tm/vue-number-format#44]
I used 2.9.0 for Vue 2, when input sometimes watch and computed not detecting. Sometimes it needs to unfocus or click outside the inputs to detect it and reflect changes.

**Comment by lexar19** [comment]
For the meantime I revert back to version 2.2.12, coz 2.5 - 2.9.0 have the same issue. Hope you able to fix this, thank you.

**Comment by dipaksarkar** [comment]
Please use latest version v2.13.0 for Vue 2

**Comment by lexar19** [comment]
Issue still occur in v2.13.0

**Comment by dipaksarkar** [comment]
Can you tell me how to i can replicate the issue?

**Comment by lexar19** [comment]
Example I have this input <input v-model="price" v-number="number">, when inputting the watch and computed sometimes not reflecting or do what suppose to do.

```
<input v-model="price" v-number="number">
<script>
watch: {
 price(){
   .. do something
 }
},
computed: {
  calculate(){
    return this.price * 10;
   }
},
</script>
```

**Comment by lexar19** [comment]
In v2.2.12 its working fine.

**Comment by lexar19** [comment]
Its not about the unmasked value, theres a reason as to why I use it as directive and I already handled the comma and any error when calculating, so the issue is not in my side.

**Comment by lexar19** [comment]
Even here https://codesandbox.io/s/vue-2-playground-forked-m6mp17?file=/src/App.vue, the issue also occur, pls compare the 2 images. The correct result is the 2nd image.
![image](https://user-images.githubusercontent.com/30932969/229438838-14fdf471-298d-4894-8ca2-e5198b20069d.png)
![image](https://user-images.githubusercontent.com/30932969/229438967-a716c963-fecd-4a62-9d80-bd35f43057c5.png)

**Comment by lexar19** [comment]
I think the issue is when deleting or using backspace.

**Comment by dipaksarkar** [comment]
I have fix the issue.
https://codesandbox.io/s/coders-tm-vue-number-format-2-m6mp17?file=/src/components/DirectiveBased.vue

**Comment by lexar19** [comment]
I still capture some issue pls see screenshot below. I found another issue in which sometimes the input reverted back. For example, I input 10,000 then 100,000 sometimes it reverts back to 10,000 or vice versa.
![image](https://user-images.githubusercontent.com/30932969/231079346-89aa93ef-5859-43e4-8fb0-fab96e9933af.png)

**Comment by dipaksarkar** [comment]
Please check now, I have published new version 2.18.0 for vue 2
https://codesandbox.io/s/coders-tm-vue-number-format-2-m6mp17?file=/src/components/DirectiveBased.vue

**Comment by lexar19** [comment]
All fix now, thank you :)
